python/ray/worker.py

```diff
@@ -710,7 +710,7 @@ def run_function_on_all_workers(self, function):
                 return
 
             check_oversized_pickle(pickled_function, function.__name__,
-                                   "function", self)
+                                   "remote function", self)
```

Comment:
Ah, this one should actually not be `"remote function"`, this one should still be `"function"`.